app/ts/store/favorites.ts

```diff
@@ -35,7 +36,7 @@ const favorites = signal<FavoriteModel[]>(getCachedFavorites())
 
 effect(() => {
 	const uniqueTxns = Array.from(new Set(favorites.value))
-	localStorage.setItem(FAVORITES_CACHE_ID, JSON.stringify(uniqueTxns))
+	localStorage.setItem(FAVORITES_CACHE_ID, JSONStringify(uniqueTxns))
```

Comment:
I feel we should use funtypes here for parsing and serializing instead of json stringify without typings. Is there a reason why you don't want to do more checking?